### PR TITLE
feat(*): support arm64

### PR DESCRIPTION
**Pull request by qianlei90**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to http://jira.caicloud.xyz/browse/CPS-5956

**Special notes for your reviewer**:

/cc @hezhizhen @whalecold 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -61,7 +61,7 @@ CMD_DIR := ./build
 BUILD_DIR := ./build
 
 # Current version of the project.
-VERSION ?= v1.4.3
+VERSION      ?= $(shell git describe --tags --always --dirty)
```
Comment (whalecold) [inline Makefile]: 这样是不是需要先打 tag 这个才会生效，顺序反了。

**Comment by hezhizhen** [comment]
LGTM

File: Makefile (hunk 1 above)
Comment (qianlei90) [reply Makefile]: 已经有tag的呀，最新的是`v1.4.4`

Comment (whalecold) [reply Makefile]: 哦哦 我想错了

**Pull request merged**